proto/stride/stakeibc/host_zone.proto

```diff
@@ -36,8 +36,6 @@ message HostZone {
       [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string community_pool_redeem_holding_address = 31
       [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
-  string community_pool_address = 32
-      [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```

Comment:
Good catch, we originally thought this was needed for the call to FundCommunityPool but it is not so we should remove it!